fhir-resource-wih/src/main/java/io/elimu/a2d2/fhirresourcewih/OperationOutcomeWorkItemHandler.java

```diff
@@ -0,0 +1,205 @@
+package io.elimu.a2d2.fhirresourcewih;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.kie.api.runtime.process.WorkItem;
+import org.kie.api.runtime.process.WorkItemHandler;
+import org.kie.api.runtime.process.WorkItemManager;
+
+import ca.uhn.fhir.context.FhirContext;
+import io.elimu.a2d2.exception.WorkItemHandlerException;
+import io.elimu.a2d2.genericmodel.ServiceResponse;
+
+public class OperationOutcomeWorkItemHandler implements WorkItemHandler {
```

Comment:
The naming convention we have used for other WIH's is ____Delegate. Should we name this one OperationOutcomeDelegate?